strings-csharp/HowToAddSeparatorsInThousandsPlaceForANumber/HowToAddSeparatorsInThousandsPlaceForANumber/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Globalization;
+
+// String.Format() Method
+
```

Comment:
@biagianina the Program.cs can be better arranged so it is easy to follow.

Can you look at this project for some pointers. 
https://github.com/CodeMazeBlog/CodeMazeGuides/tree/main/strings-csharp/ReplaceLineBreaksInAStringCSharp

Reply:
@elfico I made the changes, is the new commit sufficient? Do I need to do anything else?